dasharo-compatibility/dts-update.robot

```diff
@@ -0,0 +1,94 @@
+*** Settings ***
+Library             Collections
+Library             OperatingSystem
+Library             Process
+Library             String
+Library             Telnet    timeout=20 seconds    connection_timeout=120 seconds
+Library             SSHLibrary    timeout=90 seconds
+Library             RequestsLibrary
+Resource            ../keywords.robot
+Resource            ../keys.robot
+Resource            ../rtectrl-rest-api/rtectrl.robot
+Resource            ../sonoff-rest-api/sonoff-api.robot
+Resource            ../variables.robot
+
+Suite Setup         Run Keywords
+...                     Prepare Test Suite
+Suite Teardown      Run Keyword
+...                     Log Out And Close Connection
+# This must be in Test Setup, not Suite Setup, because of a known problem
```

Comment:
Should be already fixed: https://github.com/Dasharo/open-source-firmware-validation/issues/132#issuecomment-2063498232
Can you please verify?

Reply:
removed the unnecessary test setup [here](https://github.com/Dasharo/open-source-firmware-validation/pull/267/commits/a47ea84380a7aca7847890baa0e1e8d0406c83d7)